platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollector.java

```diff
@@ -117,6 +137,15 @@ public void run() {
       eventAdmin.postEvent(
           new Event(
               SystemNotice.SYSTEM_NOTICE_BASE_TOPIC + "insecureDefaults", notice.getProperties()));
+
+      if (Files.exists(USERS_PROPERTIES_FILE_PATH) && usersPropertiesDeletionFlag) {
```

Comment:
I removed userPropertiesAlerts because those were taking over the `users.properties` banner and users wouldn't be able to easily see what they need to do to avoid the system from shutting down.